### PR TITLE
updating tines description

### PR DESCRIPTION
**Pull request by gautambaghel**
### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by gautambaghel** [comment]
@mgarrell777 -- just updated the description, please merge whenever's convenient

File: website/docs/cloud-docs/integrations/run-tasks/index.mdx [hunk 1]
```diff
@@ -184,9 +184,9 @@ Styra DAS works with Terraform Cloud to always evaluate plans for an apply actio
...
 
 ### Tines
-Tines enable teams to do their best work by limiting time wasted on manual tasks. The Terraform Cloud pre-plan run tasks integration with Tines helps users to approve and record infrastructure requests. 
+Tines is a no-code automation platform that provides smart, powerful technology to enable teams to do their best work and limit time wasted on manual, error-prone tasks. Tines manages mission-critical workflows, automating and responding to real-time requests. With Tines and Terraform Cloud, users create and prioritize requests with a click of a button. Plus, they can provide consistent updates on the requests in the chat systems they use every day.
 
-To get started, follow the instructions in the [Terraform Cloud integration](https://www.tines.com/story-library/102061/respond-to-configure-terraform-cloud-run-task) user documentation.
+To get started, build a workflow from Terraform Cloud template or follow the instructions in the [Terraform Cloud integration](https://www.tines.com/story-library/102061/respond-to-configure-terraform-cloud-run-task) user documentation.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/integrations/run-tasks/index.mdx:189]:
```suggestion
To get started, build a workflow from a Terraform Cloud template or follow the instructions in the [Terraform Cloud integration](https://www.tines.com/story-library/102061/respond-to-configure-terraform-cloud-run-task) user documentation.
```

**Review by mgarrell777 — APPROVED** [review]
@gautambaghel Thanks for the PR. Very minor change. Looks good overall.

**Comment by gautambaghel** [comment]
@mgarrell777 - Done!

**Pull request merged**